### ISSUE TITLE
nw.-js 0.15.x problem still "loading..."

### ISSUE THREAD

**Issue by sourceskyboxer** [nwjs/nw.js#5137]
I have problem because nw.js freezes - if I use special frameworks of javascript like sproutcore ( just awesome gui libraries like Qooxdoo or ExtJs or UI-JQuery whatever... 

I am trying with sproutcore framework to my nw.js app.

If you get http://sproutcore.com/ 
than you need to download ruby ( support good with x86 because they compile without errors )
PS: Please do not use x64 files because sproutcore can not work to generate with x64 because it always outputs error like make [1] error.....

You need to use x86 installer / x86 package on OS x64 ot OS x86.

than sc-gen project getting_started
and you will test sc-server 
Load from browser localhost:4020 

If you have got error "blocked ... ::1" than you try use sc-server --allow-from-ips=*.*.*.*
Load from browser localhost:4020

if you see "label" than it works fine
sc-build --build=test_build

wait for generate standalone deployed files and index.html replace:
/static.... or / remove from path to javascript and css files...

than i add package.json and test nw .
and why does it shows still "loading ... "

How do I fix?
![nwjs_loading](https://cloud.githubusercontent.com/assets/3496136/17040158/984376f0-4f9e-11e6-8daf-0f0e9d408d6a.PNG)

**Comment by ghostoy** [comment]
@sourceskyboxer Did you see any errors in DevTools?

**Comment by sourceskyboxer** [comment]
Yes impossible error: like this copy error: module.js:466 Uncaught Error: Cannot find module 'license'

But I didn't that. I would like to sproutcore framework I have to build version of example website into local server but nw gets error . If you try sproutcore with nw.js than it doesn't work :( So sad. What I should use framework like Qooxdoo or Extjs with nw.js

**Comment by Christywl** [comment]
Is this issue still reproduced on the latest nwjs build?

**Comment by sourceskyboxer** [comment]
Why do you ask? Please check website if you have to chance if latest build too If you have to generate own web application into nw.js than it shows "@ Loading" why does it not show web application. cause nw.js doesn't load features of web application.

**Comment by Christywl** [comment]
I try this on Mac:

1. `sudo npm install -g sproutcore`
2. `sproutcore init test`
3. `sproutcore gen app nwjs`
4. `sproutcore build nwjs`
5. `sproutcore serve`
    Server starts on http://localhost:4020, it works fine
6. cd `build/static/nwjs/en/291c12a2918ef7871e664034b817a87e32039dfa` and add `package.json`
7. Launch app via nwjs:
  `/User/test/test/nwjs-sdk-v0.26.6-osx-x64/nwjs.app/Contents/MacOS/nwjs .`

<img width="1280" alt="screen shot 2017-11-23 at 3 43 26 pm" src="https://user-images.githubusercontent.com/20039821/33162561-1839356e-d065-11e7-823d-69b3a5dae620.png">

**Comment by sourceskyboxer** [comment]
That is same problem with Windows 10 too I can't fix. If sometimes frameworks have high level sandbox won't load into Chromium. I think it need disable security of sandbox. - But I am not sure..
